go.mod

```diff
@@ -106,8 +106,6 @@ require (
 	oss.terrastruct.com/util-go v0.0.0-20231101220827-55b3812542c2
 )
 
-require github.com/containerd/console v1.0.4-0.20230313162750-1ae8d489ac81
```

Comment:
@vito @sipsma I notice this keeps showing up for some reason? e.g. https://github.com/dagger/dagger/pull/6714/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6

We already have this import on line 20, so I genuinely have no idea why it keeps appearing back again? Just wanted to raise, it keeps disappearing whenever I run `go mod tidy`.

Reply:
Maybe a Go version issue? I dev with 1.21.7 at the moment.

Or maybe an artifact of local `go mod edit -replace` or `go work` usage that is uncommitted?

Maybe we should have a `go mod tidy` linter?